app/views/tags/index.html.erb

```diff
@@ -16,12 +16,19 @@
         <thead>
           <tr class="table-header">
             <th>Name</th>
+            <th></th>
           </tr>
         </thead>
         <tbody>
           <% @tags.each do |tag| %>
             <tr class="t-tag">
               <td><%= tag.name %></td>
+              <td><%= link_to 'Delete',
```

Comment:
Just had a quick chat here about 'tags' vs 'groups'. The users of this system have been shown a number of iterations of the prototype that refer to this functionality as groups...

Could we stick with that for anything in the UI? I guess it's fine to have them described as tags in the model, but their understanding on the domain language is a 'group' for this kind of thing.